cypress/Dockerfile

```diff
@@ -33,20 +33,15 @@ RUN yum install -y unzip
 
 RUN npm install yarn -g
 
-RUN mkdir -p ~/.cache/Cypress/8.3.0/Cypress && \
-    wget --no-check-certificate http://download.cypress.io/desktop/8.3.0?platform=linux -O /tmp/z.$$ && \
-    unzip -d ~/.cache/Cypress/8.3.0/ /tmp/z.$$ && \
-    rm /tmp/z.$$ && \
-    chmod +x ~/.cache/Cypress/8.3.0/Cypress/Cypress && \
-    rm -rf /var/lib/apt/lists/*
-
 COPY / hmda-frontend/
```

Comment:
should this line be changed to absolute path as well?

Reply:
I adjusted the `WORKDIR` path because my IDE was giving me a warning, something to the effect that `WORKDIR paths should be absolute`.  Since I didn't get a similar warning for the `COPY` command, I left it as is. That said, things are working as currently configured, so I think we can leave this as relative.